ert_shared/dark_storage/graphql/ensembles.py

```diff
@@ -104,8 +111,9 @@ def resolve_unique_responses(root: Any, info: "ResolveInfo") -> "UniqueResponse"
         raise NotImplementedError
 
     @staticmethod
-    def resolve_parameters(root: Any, info: "ResolveInfo") -> "Parameter":
-        raise NotImplementedError
+    def resolve_parameters(root: Any, info: "ResolveInfo") -> List[dict]:
+        res = get_res()
+        return res.get_ensemble_parameters("whatever")
```

Comment:
Is the parameter here needed ( `whatever` ). I cannot see that the `ensemble_id` is used on the other side?

Reply:
i think root gives you the case/ensemble-name, by changing your implementation in the facade to this
```
    def get_ensemble_parameters(self, ensemble_name: str) -> List[dict]:
        return [
            dict(
                name=key,
                values=parameter.values.tolist(),
            )
            for key, parameter in (
                (key, self.gather_gen_kw_data(ensemble_name, key))
                for key in self.all_data_type_keys()
                if self.is_gen_kw_key(key)
            )
        ]
```

you will get the parameters for the correct case. I think.